esp/xslt/esdl2monitor.xslt

```diff
@@ -203,32 +204,47 @@ END;
     string100 Message {xpath('Message')};
   END;
 
-  soapoutRec := record
+  soapoutRec := record <xsl:choose><xsl:when test="$platform='ESP'">(the_responseLayout)
```

Comment:
The goal is to get the generated code to be formatted right, more than the xslt.  This form of xslt makes it clear to me that the generated code is definitely going to be one line.
`soapoutRec := record (the_responseLayout)`
not
`soapoutRec := record 
  (the_responseLayout)`
or whatever.